### DISCUSSION TITLE
What is the Pi OS that I should flash on

### DISCUSSION THREAD

**Discussion by spookstream** [iot-root/gardyn-of-eden#80] [Q&A]
My friend bought a pre-used planter that is using a raspery pi 4, with this project on it. Unfortunately, we suspect that there is a software error and we need to reflash everything on a new sd card. 

Please forgive my ignorance when I ask this question, I just bought a pi 4, 2 days ago to help her with this issue. But, in the pi imager what OS Version of ubuntu should I install as the base?  

The Getting Started guide only says a fresh install of Linux, but I have been trying to figure out witch one. 

![pi imager ubuntu options](https://github.com/user-attachments/assets/dced1626-b222-4967-bb83-53c02ac9976a)

**Comment by iot-root** [comment]
For the pi4, I would be concerned about power draw if using the gardyn control board, but you might be okay there. That said, I would do Raspian Pi 64 lite. For ubuntu , do server 64bit. I stay away from the bloat of the desktop since you dont need a GUI.

For the pi zero and zero 2 I do 32bit-lite

Hope that helps.
